mapstruct-kotlin/src/main/kotlin/de/kevcodez/Main.kt

```diff
@@ -0,0 +1,20 @@
+package de.kevcodez
+
+import de.kevcodez.converter.PersonConverterImpl
+import de.kevcodez.model.Person
+import java.time.LocalDate
+
+fun main(args: Array<String>) {
+
+    val converter = PersonConverterImpl()
```

Comment:
This is aligned with the comment in the `README`. The comment is just as a reminder.